tests/api_validation_test.go

```diff
@@ -322,7 +325,7 @@ var _ = Describe("[rfe_id:1130][crit:medium][posneg:negative][vendor:cnv-qe@redh
 			By(fmt.Sprintf("Waiting for datavolume to match phase %s", string(cdiv1.ImportInProgress)))
 			err = utils.WaitForDataVolumePhase(f, f.Namespace.Name, cdiv1.ImportInProgress, dataVolumeName)
 			if err != nil {
-				f.PrintControllerLog()
+				f.PrintControllerLog(testBegin)
```

Comment:
I think `PrintControllerLog` isn't really needed anymore, from a quick check, it dates back 6 years
And we also have these type of logs under artifacts
https://gcsweb.ci.kubevirt.io/gcs/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/3285/pull-containerized-data-importer-e2e-ceph/1793274221523111936/artifacts/1/pods/